Popup/explainer.md

```diff
@@ -290,21 +315,79 @@ Freedom over the size and position of a `popup` could enable an author to spoof
 
-## Areas for exploration
+4. **Show/hide or show/close?** This proposal introduces a symmetrical `show`/`hide` method pair on the proposed `popup` element. `dialog`, however, sets a precedent for a `show` and `close` method pairing. That seemed less intuitive, but perhaps the existing pattern should be followed. Alternatively, the platform could introduce `hide` on `dialog` and consider `close` deprecated.
 
-* **Showing `popup`s declaratively:** `popup` requires authors to call `show()` in script, which is not ideal for authors who wish to work declaratively. We may consider an HTML attribute that would show a relevant `popup` when the element the attribute is applied to is invoked.
+## Areas for exploration
 
 * **Focus trapping:** the [`inert` attribute](https://whatpr.org/html/4288/interaction.html#the-inert-attribute) enables authors to mark portions of a document as inert, such that user input events are ignored. Inverting this model, new primitives could enable focus trapping with parts of a document, e.g. a `popup`. New focus trapping primitives could be useful in cases where the tab cycle should be constrained to the `popup`, but the rest of the document would receive other types of user input.
 
 * **Animating state transitions:** applying animations and transitions to interactive elements’ show/hide states can be difficult. For example, to apply a CSS transition the element must first produce an initial box before its properties can be transitioned to new values. That requires a two step process to first show the element, and in a subsequent frame, initiate a transition by applying a class. Likewise, since the browser manages the visibility of the popup for light dismiss behaviors, it is impossible to apply a close animation. To address this issue perhaps the answer is to invent a new CSS animation primitive that is triggered when an element stops producing a box.
+
+## Appendix
+
+### Anchoring and event bubbling
+
+In a previous version of this document, we proposed that the hierarchy created by the `anchor` attribute relationship affects the event propagation path. With the introduction of a separate `popup` attribute which creates an invocation relationship, it is less clear whether event bubbling should be applied as a result of the `popup` attribute and/or the `anchor` attribute.
+
+This behavior as previously proposed adds complexity to the platform, and it is not clear whether there is enough value to authors for the platform to take on that complexity. We welcome feedback on this point and preserve the previous proposal here.
+
+#### Example of event bubbling

```

Comment:
For completeness sake, I wonder if we should also mention what the author would need to do if we don't have modified event bubbling.

E.g. could note that the line `e.currentTarget.dispatchEvent(new CommandEvent(e.currentTarget.id))`
would have to instead be something like 
`bugCommands.anchor.dispatchEvent(new CommandEvent(e.currentTarget.id))`.

Reply:
Thanks Dan! Lemme know if this works for ya: https://github.com/MicrosoftEdge/MSEdgeExplainers/pull/459/commits/9a8a073ef2c211685980532c77bc91d5234ce8c7

Reply:
Looks great, thanks!